README.md

```diff
@@ -1,3 +1,8 @@
+## **NOTE**
+> This version of Gatsby v5.0.5 is for a specific customer only 
+
+
+
```

Comment:
please get the note verified from docs don't add specific customer @Contentstack-AnkitaD